editor/editor_resource_preview.cpp

```diff
@@ -138,6 +140,10 @@ void EditorResourcePreview::_generate_preview(Ref<ImageTexture> &r_texture, Ref<
 	if (type.is_empty()) {
 		r_texture = Ref<ImageTexture>();
 		r_small_texture = Ref<ImageTexture>();
+
+		if (is_print_verbose_enabled()) {
+			print_line(vformat("Generated '%s' preview in %d usec", p_item.path, OS::get_singleton()->get_ticks_usec() - started_at));
+		}
```

Comment:
We have `print_verbose` for this:
```suggestion
		print_verbose(vformat("Generated '%s' preview in %d usec", p_item.path, OS::get_singleton()->get_ticks_usec() - started_at));
```

Reply:
> Note that I couldn't use `print_verbose` here because `print_verbose` is undefined by the included `variant_utility.h` header. Probably something worth addressing?

I had missed this comment. So indeed this can't be done for now.